### PR TITLE
exchange heapster status

### PR DESCRIPTION
**Pull request by zouyee**
**What this PR does / why we need it**:
remove heapster description using metrics-server instead.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Comment by andyxning** [comment]
@zouyee Although heapster has been retired, but i can be sure that some people or companies are still using it. Remove the description about heapster completely is not so good. How about we just exchange the position about heapster and metrics-server, as for now, metrics-server is the recommended way to collect metrics for Kubernetes instead of heapster. :) Just the opposite before heapster is retired.

**Comment by andyxning** [comment]
@zouyee You have not change according to https://github.com/kubernetes/kube-state-metrics/pull/614#issuecomment-446074778. :)

**Comment by andyxning** [comment]
@zouyee Do you still have some time to update this?

**Comment by zouyee** [comment]
done! @andyxning

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ the raw metrics.
 - [Metrics Documentation](#metrics-documentation)
 - [Kube-state-metrics self metrics](#kube-state-metrics-self-metrics)
 - [Resource recommendation](#resource-recommendation)
-- [kube-state-metrics vs. Heapster(metrics-server)](#kube-state-metrics-vs-heapstermetrics-server)
+- [kube-state-metrics vs. metrics-server(Heapster)](#kube-state-metrics-vs-heapstermetrics-server)
```
Comment (andyxning) [inline README.md]: Link is broken. @zouyee

File: README.md [hunk 2]
```diff
@@ -132,32 +132,32 @@ These numbers are based on [scalability tests](https://github.com/kubernetes/kub
...
 
 In other words, kube-state-metrics itself is designed to be another source for
-Heapster([metrics-server](https://github.com/kubernetes-incubator/metrics-server)) (although this is not currently the case).
+[metrics-server](https://github.com/kubernetes-incubator/metrics-server)(Heapster) (although this is not currently the case).
 
-Additionally, some monitoring systems such as Prometheus do not use Heapster([metrics-server](https://github.com/kubernetes-incubator/metrics-server))
+Additionally, some monitoring systems such as Prometheus do not use [metrics-server](https://github.com/kubernetes-incubator/metrics-server)(Heapster)
 for metric collection at all and instead implement their own, but
-[Prometheus can scrape metrics from heapster itself to alert on Heapster(metrics-server)'s health](https://kubernetes.io/docs/tasks/debug-application-cluster/core-metrics-pipeline/).
+[Prometheus can scrape metrics from heapster itself to alert on metrics-server(Heapster)'s health](https://kubernetes.io/docs/tasks/debug-application-cluster/core-metrics-pipeline/).
```
Comment (andyxning) [inline README.md]: `from heapster itself` -> `from metrics-server(Heapster)`

**Pull request merged**